### ISSUE TITLE
Add date to downloads somewhere, pls

### ISSUE THREAD

**Issue by FredFoo** [vmware/photon#973]
If you are waiting for a new release you might go and check https://github.com/vmware/photon/wiki/Downloading-Photon-OS for updates. But there is no info there. Then you think you are clever, go to https://github.com/vmware/photon/releases - hmpf, last update March 2019. Then you try the actual download page by cutting off the filename from the url http://dl.bintray.com/vmware/photon/3.0/Rev2/iso - hmpf, only filenames, no file details.

Did I miss the changelog? The release file?

I see there is a commit hash in the iso filename, but that only tells me if the version on your server is different from the version that I have.

But if for instance I find an issue that is fresh, but closed and a fix is submitted and I want to figure out if there is even a chance the current release includes it - hmpf, no chance. 

Any hint? Plan?

**Comment by dcasota** [comment]
Bump on this - the latest releases or tags on https://github.com/vmware/photon/releases are not listed, at least not visible for any github users. Well, it should, according to https://vmware.github.io/photon/assets/files/html/3.0/photon_troubleshoot/checking-the-version-and-build-number.html. You can find the build number of PhotonOS using ```cat /etc/photon-release```.

https://bintray.com/vmware/photon, see 3.0/Rev2, shows up a version publication date. For a tdnf distro-sync ~changelog, a date range correlation with https://github.com/vmware/photon/commits/master may help .

**Comment by ecerichter** [comment]
Seems vmware gave up on Photon, since no queries are being answered anywhere...

**Comment by tompscanlan** [comment]
Sorry folks.  Photon isn't given up, just a busy time of year for VMware people. @YustasSwamp  has been closing some issues recently, so hoping he'll catch up to here soon.

I'm in vmware, but not focused on this project, just stopping by.

@FredFoo , since your issue creation, there was a new release added to the page https://github.com/vmware/photon/wiki/Downloading-Photon-OS with some dates added: [see the diff](https://github.com/vmware/photon/wiki/Downloading-Photon-OS/_compare/bfe04a02402e8fc401e30786c0ddf27e3c7650f0...0d4c3eb0919b52d8f8022024786d9028ad75c66b)

Does that fix you up for now?  It does sound like the release process needs better automation around documenting at release time for a permanent fix.

**Comment by YustasSwamp** [comment]
Hi @FredFoo, I got your concern,

https://github.com/vmware/photon/wiki/Downloading-Photon-OS
The download page is a source of information.
It has up to date information about Photon OS releases (GA, revs) of deliverables such as ISO, ova. We do not publish them frequently.
If you take a look on download link of any file, you will see the corresponding commit hash (**58f9c74** as example)
http://dl.bintray.com/vmware/photon/3.0/Rev2/iso/photon-3.0-58f9c74.iso
https://github.com/vmware/photon/commit/58f9c74

We do not create tags/releases for packages updates anymore as there is no consumers.
Instead we publish [security advisories](https://github.com/vmware/photon/wiki/Security-Advisories) for all updates in all branches. This is what get traction. And people/teams consume this information instead. These updates are pretty frequent. For instance, for Photon 3.0 it is ~2 times a week: https://github.com/vmware/photon/wiki/Security-Advisories-3

We will consider using release/tags for Photon OS releases such as 4.0 and so on.

Thanks!

**Comment by FredFoo** [comment]
@YustasSwamp : I just updated a VM with Photon 3 and again have no clou what I got. And when I try to find out I am lost. That build number in /etc/photon-release is just using disk space it seems. What can I do with that? What is it even supposed to say?

Why dont you add a highwatermark to the file? Just the most recent git commit id, the timestamp it was committed and the branch it was commited to? Every package updates this information if its commit timestamp is later than the current one. 

This way I could study the git log and see which changes I have and what has not reached me yet. 

Or at least add the last commit of the release to the build number - if that would give me a clou.

I did look at the security advisories just now. There was a new version of containerd - 1.3.7. I check on my freshly updated system and see I have 1.2.14. Makes me think I might not have the latest minor release. And makes me wonder where I can find out what is the latest minor release and which one do I have installed? I am sure its in there somewhere and I just have trouble finding it. Right?
